src/js/components/Button/StyledButton.js

```diff
@@ -27,8 +24,7 @@ const primaryStyle = props => css`
   ${
     backgroundStyle(
       normalizeColor(
-        props.colorValue || props.theme.button.primary.color
```

Comment:
why are we removing `brand` ?

Reply:
this seems like a potential unexpected change

Reply:
I've added a note in the pull request description.